### PR TITLE
Web Main - Alpha 0.1 Basic functionality

### PR DESCRIPTION
**Pull request by vokymir**

### CODE REVIEW

File: spot/cli/server.py [hunk 1]
```diff
@@ -8,3 +10,70 @@ def test_handler():
 def run_http_server():
     HttpServer.add_handle("/cli", test_handler)
     HttpServer.run(host="0.0.0.0", port=4321)
+
+# ------- Handling HTTP requests from client -------
+
+@app.route("/api/movement", methods=['POST'])
+def handle_post_request_movement():
+    if request.method == 'POST':
```
Comment (Maneren) [inline spot/cli/server.py]: Please convert to guard conditions

File: website/assets/communication.js [hunk 2]
```diff
@@ -0,0 +1,138 @@
+
+setInterval(sendAll, 1000);
```
Comment (Maneren) [inline website/assets/communication.js]: Send the state on change, not periodically

**Review by Maneren — CHANGES_REQUESTED** [review]
Good job. Don't spam

**Review by vokymir — COMMENTED** [review]
fixed

**Review by Maneren — APPROVED** [review]
great

**Pull request merged**